### PR TITLE
Feature/story links from news items

### PR DESCRIPTION
**Pull request by not4Pedro**
Add button to save to story all the links available in the story. Possibility to add link manually too.

depends on #316

## Summary by Sourcery

Add functionality to manage story links by introducing a new StoryLinks component and updating the Story model to include a 'links' attribute. Implement a database migration to support the new links feature.

New Features:
- Introduce a new component, StoryLinks, to manage and display links associated with a story, allowing users to add links manually or update them from news items.

Enhancements:
- Extend the Story model to include a new 'links' attribute, enabling the storage and management of links related to a story.

Build:
- Add a new database migration script to include a 'links' column in the story table, facilitating the storage of links in JSON format.

### CODE REVIEW

File: src/core/core/model/story.py [hunk 1]
```diff
@@ -39,6 +40,7 @@ class Story(BaseModel):
     comments: Mapped[str] = db.Column(db.String(), default="")
     summary: Mapped[str] = db.Column(db.Text, default="")
     news_items: Mapped[list["NewsItem"]] = relationship("NewsItem")
+    links: Mapped[list["StoryLink"]] = relationship("StoryLinks", back_populates="story", cascade="all, delete-orphan")
```
Comment (b3n4kh) [inline src/core/core/model/story.py:43]: This is overly complicated.


Switch it to:     `links: Mapped[list[str]] = db.Column(JSON)`

There you could either store a list of links, or a list of dict [{ news_item_id: link }]

File: src/gui/src/components/assess/StoryLinks.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
+<template>
+  <v-combobox
```
[the remark this answers is not in the document]
Comment (b3n4kh) [reply src/gui/src/components/assess/StoryLinks.vue:2]: Worth considering, but not necessary

**Review by b3n4kh — CHANGES_REQUESTED** [review]

**Review by b3n4kh — APPROVED** [review]

File: src/gui/src/components/assess/StoryLinks.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (not4Pedro) [reply src/gui/src/components/assess/StoryLinks.vue:2]: This is a good direction, but for now, I would say it's sufficient how it is

**Review by b3n4kh — CHANGES_REQUESTED** [review]

**Comment by not4Pedro** [comment]
@sourcery-ai review

**Pull request merged**